Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Company.FunctionApp.csproj

```diff
@@ -11,13 +11,13 @@
   </PropertyGroup>
 <!--#if (Framework == "net6.0")-->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.6.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.8.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.3.0" />
   </ItemGroup>
 <!--#endif -->
 <!--#if (Framework == "net7.0")-->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.6.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.8.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.7.0-preview2" />
   </ItemGroup>
 <!--#endif -->
```

Comment:
You need to update the version for net framework too on line 26

1.8.0 is ahead of 1.8.0-preview3